extensions/wikia/VisualEditorTourExperiment/scripts/VisualEditorTourExperimentInit.js

```diff
@@ -3,25 +3,24 @@ define('VisualEditorTourExperimentInit',
 	function ($, VETour, veTourConfig, abTest, tracker) {
 		'use strict';
 
-		var experimentName = 'contribution-experiments',
-			freshlyRegisteredExperimentId = 5654433460,
-			usersWithoutEditExperimentId = 5735670451;
-
+		var experimentName = 'contribution-experiments';
+		
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3ATrailingWhitespace)